### PR TITLE
Benchmark suite updates to ibench, modification to gflop default behavior.

### PR DESCRIPTION
**Pull request by triskadecaepyon**
Adds a basic suite of benchmarks to the ibench project.  Also changes gflop default behavior to be optional on the reporting.

### CODE REVIEW

File: ibench/benchmarks/bench.py [hunk 1]
```diff
@@ -19,6 +19,7 @@ def _log(self, message):
     def _run(self, n):
         self._make_args(n)
 
+        # Checking on the garbage collector
```
Comment (rscohn2) [inline ibench/benchmarks/bench.py]: Change to: Disable garbage collector

File: ibench/benchmarks/bench.py [hunk 2]
```diff
@@ -34,28 +35,41 @@ def _run(self, n):
...
+    def measure(self, n, args):
         self._log('')
         self._log('  N = %d' % n)
 
         times = self._run(n)
 
         ops = self._ops(n)
-        for elapsed in times:
-            self._log('  elapsed %f gflops %f' % (elapsed,ops/elapsed))
+        if 'gflops' in args:
```
Comment (rscohn2) [inline ibench/benchmarks/bench.py]: I don't understand the logic. If gflops is in args, it doesn't print gflops? Should it be nogflops?

File: ibench/benchmarks/bench.py (hunk 1 above)
Comment (triskadecaepyon) [reply ibench/benchmarks/bench.py]: Addressing in my next commit.

File: ibench/benchmarks/bench.py (hunk 2 above)
Comment (triskadecaepyon) [reply ibench/benchmarks/bench.py]: Fixed in commit 1e819a4.

**Pull request merged**